ROADMAP.md

```diff
@@ -0,0 +1,118 @@
+
+
+# go-libp2p roadmap Q4’22/Q1’23
+
+```
+Date: 2022-10-07
+Status: In Progress
+Notes: This document is still in review will may be heavily modified based on stakeholder feedback. Please add any feedback or questions in:
+https://github.com/libp2p/go-libp2p/issues/1806
+```
+
+## 🛣️ Milestones
+### 2022
+
+#### Early Q4 (October)
+-   A.1⚡Handshakes at the Speed of Light - **[Early Muxer Negotiation](https://github.com/libp2p/specs/issues/426)**
+- E.1 🧪 Future-proof testing - **RTT test for Early Muxer Negotiation**
+
+### Mid Q4 (November)
+-   F.1 ⏱ Future-proof Benchmarking - **[Using nix-builders](https://github.com/testground/testground/pull/1425)**
```

Comment:
I think before we spin up benchmarking, lets get interoperability nailed.  The pieces are in place for that already with libp2p/test-plans. I discussed this more below.